backend/vaa-strapi/src/api/app-setting/content-types/app-setting/schema.json

```diff
@@ -12,51 +12,54 @@
   },
   "pluginOptions": {},
   "attributes": {
-    "publisherName": {
-      "type": "json"
-    },
-    "publisherLogo": {
-      "type": "media",
-      "multiple": false,
-      "required": false,
-      "allowedTypes": [
-        "images"
-      ]
-    },
-    "publisherLogoDark": {
-      "type": "media",
-      "multiple": false,
-      "required": false,
-      "allowedTypes": [
-        "images"
-      ]
-    },
-    "customData": {
-      "type": "json"
-    },
-    "poster": {
-      "type": "media",
-      "multiple": false,
-      "required": false,
-      "allowedTypes": [
-        "images"
-      ]
-    },
-    "posterCandidateApp": {
-      "type": "media",
-      "multiple": false,
-      "required": false,
-      "allowedTypes": [
-        "images"
-      ]
-    },
     "underMaintenance": {
       "type": "boolean",
       "default": false
     },
     "allowOverwrite": {
       "type": "boolean",
       "default": false
+    },
+    "header": {
```

Comment:
All the settings sections are (repeatable) components so that we get a similar hierarchical structure for them as in the frontend. Their contents are also sometimes complex, which can only be acchieved with components.

Reply:
Yeah, we can't have multi-level hierarchies because components cannot be nested within each other, but I think that's an acceptable limitation.